src/docs/tune-durability.dox

```diff
@@ -66,6 +66,40 @@ most durability guarantees).
 	logging configured; updates durable on application or system
 	failure}
 </table>
+
+The durability setting can also be controlled directly on a per-transaction
+basis via the WT_SESSION::commit_transaction method.
+The WT_SESSION::commit_transaction supports several durability modes with
+the \c sync configuration that override the connection level settings.
+
+If \c sync=on is configured then this commit operation will wait for its
+log records, and all earlier ones, to be durable on disk before returning.
```

Comment:
I fixed it for the commit part and added a sentence on `log_flush` to say it overrides that and forces an `fsync`.  Please review.